src/Sample.xoo

```diff
@@ -10,3 +10,4 @@ public class Sample1 {
     System.out.println("hello" + " xoo");
   }
 }
+// xoo
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Has Tag ([xoo:HasTag](http://127.0.0.1:17355/coding_rules#rule_key=xoo%3AHasTag))

[See it in SonarQube](http://127.0.0.1:17355/project/issues?id=sample&branch=feature2&issues=AWCXJgM-5-y_UVe1D1mF&open=AWCXJgM-5-y_UVe1D1mF)

[ISSUE_KEY]: AWCXJgM-5-y_UVe1D1mF